tests/e2e/script_test.go

```diff
@@ -265,6 +271,40 @@ func TestE2EScript(t *testing.T) {
 	})
 }
 
+// applies objects directly to the cloud provider
```

Comment:
Could we instead apply and then abandon them?  Even if we use a custom verb to do that, this feels like it's duplicating our direct controller layer (?)

Reply:
We could do that but technically we will be testing the acquisition or adoption of a resource that we created and then un-managed. As opposed to to acquiring a previsouly unknown resource to us. Although I am not convinced that this distinction is meaningful That is, we can probably use the scenario you proposed to better effect.

If we do want to acquire a "clean" resource, I can expose the adapter or reuse some of the ResourceContext methods I added in #1768